apps/frontend/src/libs/modules/notifier/notifier.ts

```diff
@@ -0,0 +1,10 @@
+import { toast } from "react-toastify";
+
+const notifier = {
```

Comment:
this should be a class, similar to Store module 
```suggestion
class ToastNotification { 
```

Reply:
Can I name the class as `ToastNotifier` instread of `ToastNotification`?

Reply:
yes, you can